### ISSUE TITLE
ChangeLog not up to date

### ISSUE THREAD

**Issue by camsaul** [dakrone/cheshire#114]
I was trying to see what's new in 5.7.1 but it looks like 5.6.3 is the last version mentioned in the [ChangeLog](https://github.com/dakrone/cheshire/blob/master/ChangeLog.md)

**Comment by dakrone** [comment]
You're right, I've been very lax with it :(

For now, until it gets updated, you'll have to look through the commit log to see what's changed (sorry about that)

**Comment by lread** [comment]
@dakrone, commenting 8 years late, but I also find the missing details a bit tricky when trying to learn what is in a cheshire release.

I could do some forensics and fill out the missing pieces and get us back into a good place.

If so, would you be open to a few tweaks in formatting as well?

Perhaps a release title of, for example:

## v5.13.0 - 2024-04-01

I'm thinking if every release is described we don't need the existing "Changes between vx and vy" titles.

And I find it helpful if the changelog items link to issue else pr else commit where practical.
Also, I like to credit the person who made the change with a link to their github profile.

Here's an example of a changelog I maintain: https://github.com/clj-commons/rewrite-clj/blob/main/CHANGELOG.adoc

The "commit log" links are part of automation on release (not sure if you are interested in that as well). I just noticed that maybe [some git version tags are missing](https://github.com/dakrone/cheshire/tags)? Automation might help with that too.

Anyhoo, lemme know if you want some free housekeeping here.

**Comment by dakrone** [comment]
Certainly, as I mentioned, I've not been very good about it, and certainly not about enforcing it for PRs. If you'd like to repair it I certainly would appreciate that.

**Comment by lread** [comment]
Happy to help.

**Comment by lread** [comment]
@dakrone can you add the missing `5.13.0` github version tag?

The [released pom](https://repo.clojars.org/cheshire/cheshire/5.13.0/cheshire-5.13.0.pom) shows it for sha: `33e075443435e6e31bd4ecedabde2ef4dce062fc`

**Comment by dakrone** [comment]
Yep, I've pushed it.
